staging/src/k8s.io/apiserver/pkg/admission/configuration/mutating_webhook_manager.go

```diff
@@ -85,9 +85,14 @@ func mergeMutatingWebhookConfigurations(configurations []*v1beta1.MutatingWebhoo
 	// webhooks in serial, they are sorted here to have a deterministic order.
 	sort.SliceStable(configurations, MutatingWebhookConfigurationSorter(configurations).ByName)
 	accessors := []webhook.WebhookAccessor{}
+	names := map[string]int{}
```

Comment:
move `names` inside the configurations loop so it only dedupes between webhooks in the same config... we don't want a change to MutatingWebhookConfiguration `a` affecting the uids for MutatingWebhookConfiguration `b`

Reply:
gah, thanks for noticing!